swagger_parser/.gitignore

```diff
@@ -2,4 +2,4 @@
 .packages
 build/
 pubspec.lock
-.idea/
```

Comment:
And such changes are also not really needed

Reply:
> And it is also necessary to upgrade the version to 0.3.1 in pubspec.yaml after the fixes.

Done.